README.md

```diff
@@ -204,16 +204,17 @@ Klaviyo.createEvent(event)
 
 ### Setup
 The Klaviyo Push SDK for Android works as a wrapper around `FirebaseMessagingService`, so the
-setup process is very similar to the Firebase client documentation linked above.  
-In your `AndroidManifest.xml` file, register `KlaviyoPushService` to receive `MESSAGING_EVENT` intents.
+setup process is very similar to the Firebase client documentation linked above.  We've added it to the library
+manifest, which will be merged with your app on build. If you'd like to override this implementation with your 
+own push service, consider it will impact SDK functionality.
 
 ```xml
 <!-- AndroidManifest.xml -->
 <manifest>
     <!-- ... -->
     <application>
         <!-- ... -->
-        <service android:name="com.klaviyo.pushFcm.KlaviyoPushService" android:exported="false">
```

Comment:
since we already have advanced setup instructions that refer to implementing your own subclass, I'd recommend just removing the manifest code sample from the basic setup instructions entirely.